app/retail/templates/idea_template.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n static %}
+<script id="idea_template" type="text/x-jsrender">
```

Comment:
Can we fix the indentation to 2-spaces in this file?

Reply:
yes, format doesn't work here :rofl: